pyetrade/order.py

```diff
@@ -10,13 +10,68 @@
 
 """
 
+import dateutil.parser
 import logging
 import jxmlease
+from nose.tools import *
+import pdb

```

Comment:
Do we need these two imports `pdb`, `nose.tools`?

Reply:
nose.tools are needed for the `assert_equal`, which prints the values on failure, so is more informative than the plain `assert`.

`pdb` is needed for debugging, I think it's better than add & remove this import every time after each debug session; and it is in the python std built-in lib, which means zero dependencies.

Reply:
These imports are no longer used. Unused imports should not be left in prod code and linters will call them out. It is better to add and remove pdb as needed because this is only used in the dev lifecycle. Most the time when using pdb a one liner will be used like `import pdb; pdb.set_trace()` to enter at a specific point. Also, not ever dev will use PDB. I know pycharm and other IDEs have their own debugger.

Reply:
pdb removed.

and this changed to:

from nose.tools import assert_equal

is needed in the option_symbol()